docs/pages/enroll-resources/desktop-access/active-directory.mdx

```diff
@@ -28,6 +28,7 @@ To complete the steps in this guide, verify your environment meets the following
   encrypted LDAP connection, you should verify that your domain uses Active Directory
   Certificate Services or a non-Microsoft certification authority (CA) to issue LDAPS
   certificates.
+- Each Domain Controller must have a valid certificate with Smart Card Logon EKU included
```

Comment:
Note to self: we should probably link to https://learn.microsoft.com/en-us/troubleshoot/windows-server/certificates-and-public-key-infrastructure-pki/enabling-smart-card-logon-third-party-certification-authorities here.

Reply:
Added